File: backend/src/main/java/com/williamdsw/semsys/domain/dto/StateDTO.java
package com.williamdsw.semsys.domain.dto;

import java.io.Serializable;
import com.williamdsw.semsys.domain.State;

public class StateDTO implements Serializable
{
	// FIELDS
	
	private static final long serialVersionUID = 1L;
	private Integer id;
	private String name;
	private String abbreviation;
	
	// CONSTRUCTORS
	
	public StateDTO () {}
	public StateDTO (State state) 
	{
		super ();
		this.id = state.getId ();
		this.name = state.getName ();
		this.abbreviation = state.getAbbreviation ();
	}

	// GETTERS / SETTERS

	public Integer getId () 
	{
		return id;
	}

	public void setId (Integer id) 
	{
		this.id = id;
	}

	public String getName () 
	{
		return name;
	}

	public void setName (String name) 
	{
		this.name = name;
	}
	
	public String getAbbreviation () 
	{
		return abbreviation;
	}
	
	public void setAbbreviation (String abbreviation) 
	{
		this.abbreviation = abbreviation;
	}
}

File: frontend/src/app/models/domain/dto/state.dto.ts

export class StateDTO {

    private id: number;
    private name: string;
    private abbreviation: string;

    constructor() {}

    public getId(): number {
        return this.id;
    }

    public setId(id: number) {
        this.id = id;
    }

    public getName(): string {
        return this.name;
    }

    public setName(name: string) {
        this.name = name;
    }

    public getAbbreviation(): string {
        return this.abbreviation;
    }

    public setAbbreviation(abbreviation: string) {
        this.abbreviation = abbreviation;
    }
}
